process/datadog_checks/process/process.py

```diff
@@ -260,11 +260,11 @@ def psutil_wrapper(self, process, method, accessors, try_sudo, *args, **kwargs):
             except psutil.AccessDenied:
                 self.log.debug("psutil was denied access for method %s", method)
             except psutil.NoSuchProcess:
-                self.warning("Process %s disappeared while scanning", process.pid)
```

Comment:
What's the rationale behind this change?

Reply:
My thought was that each of the `except`s would have the same level of log output. Should I keep it as `warning`?